src/types/st-object.ts

```diff
@@ -56,6 +113,19 @@ class STObject extends SerializedType {
     const list: BytesList = new BytesList();
     const bytes: BinarySerializer = new BinarySerializer(list);
 
+    Object.keys(value).forEach((f) => {
```

Comment:
What does `f` represent? `field`?

Reply:
Yeah, `f` is the string name of the field. Will go back and pick a new name.